portfolio/src/main/webapp/script.js

```diff
@@ -17,9 +17,57 @@
  */
 
 // Spotify API keys to be used for later
-// var client_id = '444dda318d8949cb9282d95a9885a7c6';
-// var client_secret = '9f81ba6cf5748ef9d24a7ddb1e8bc4c';
-// var redirect_uri = 'https://8080-dot-12317386-dot-devshell.appspot.com/misc.html';
+var oauth_token = 'BQCebg7W1ZsfuUCdCjdcHz-GDJFMj5GjOhTxvrpNFhu3gvEzYftF3UrSIZ9KHYXVjVdF346qrcaQtd4df7FN0p5sQj2x2p3HpEsCtIDPg4owl4aZx5HfQ4wqWKY3y8l2KakUPS2ZwGKW_PbiPefFMJ3y2FVGfzmz2Kb--w9DxyLBOA';
```

Comment:
It's preferred to use `const` or `let` instead of `var`. Please replace throughout this file.

Also camelCase is preferred to snake, so please convert oauth_token to oauthToken.

As a larger point for discussion, this auth token is probably going to expire. In general, you don't want to submit code with an auth token included.